xml/adm_sudo.xml

```diff
@@ -0,0 +1,611 @@
+bash: /proc/1/maps: Permission denied
+</screen>
+
+        <para>
+          Only the <command>echo</command>/<command>cat</command> binary is run with elevated privileges,
+          while the redirection is performed by the user's shell.
+
+          You can either start a shell like in <xref linkend="sec.sudo.shell"/>
+          or use the <command>dd</command> utility instead:
+        </para>
+
+        <screen>
+echo s | sudo dd of=/proc/sysrq-trigger
+sudo dd if=/proc/1/maps | cat <!-- TODO: Better example! -->
+</screen>
+      </tip>
+    </sect2>
+
+    <sect2 xml:id="sec.sudo.shell">
+      <title>Starting a shell</title>
+      <para>
+        Having to prepend &sudo; to every command can be cumbersome.
+        While you can specify a shell as a command: <command>sudo bash</command>,
+        there are two options specifically for this purpose.
+      </para>
+
+      <variablelist>
+        <varlistentry>
+          <term>
+            <literal>sudo -s (&lt;command&gt;)</literal>
+          </term>
+          <listitem>
+            <para>
+              Starts a shell specified by the <envar>SHELL</envar> environment variable or the target user's settings.
+              If a command is given, it is passed to the shell (with the <option>-c</option> option),
+              else the shell is run in interactive mode.
+            </para>
+            <screen>
+<prompt>&exampleuser_plain;:~ &gt; </prompt>sudo -i
+root's password:
+<prompt>root:/home/tux # </prompt>exit
+<prompt>&exampleuser_plain;:~ &gt; </prompt>
+</screen>
+          </listitem>
+        </varlistentry>
+
+        <varlistentry>
+          <term>
+            <literal>sudo -i (&lt;command&gt;)</literal>
+          </term>
+          <listitem>
+            <para>
+              Like <option>-s</option>, but start the shell as login shell.
+              This means that the shell's start-up files (<filename>.profile</filename> etc.) are read
+              and the current working directory is set to the target user's <envar>HOME</envar>.
+            </para>
+            <screen>
+<prompt>&exampleuser_plain;:~ &gt; </prompt>sudo -i
+root's password:
+<prompt>root:~ # </prompt>exit
+<prompt>&exampleuser_plain;:~ &gt; </prompt>
+            </screen>
+          </listitem>
+        </varlistentry>
+      </variablelist>
+    </sect2>
+
+    <sect2 xml:id="sec.adm.sudo.usage.env">
+      <title>Environment variables</title>
+      <para>By default, &sudo; does not propagate environment variables:</para>
+      <screen>
+&prompt.user;ENVVAR=test env | grep ENVVAR
+ENVVAR=test
+&prompt.user;ENVVAR=test sudo env | grep ENVVAR
+root's password:
+<co xml:id="co.sudo.env.novar"/>
+&prompt.user;
+      </screen>
+      <calloutlist>
+        <callout arearefs="co.sudo.env.novar">
+          <para>
+            The empty output shows that the environment variable <envar>ENVVAR</envar>
+            did not exist in the context of the command run with &sudo;.
```

Comment:
It may exist in the future again, if configured. The ```did``` is there to reference **only** the example.